### PR TITLE
feat(CPH): add a new dataSource to get phone flavors

### PR DESCRIPTION
**Pull request by hmbig2**
**What this PR does / why we need it**:

1.  add a new dataSource to get phone flavors

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [X] Tests added/passed.
* [X] Documentation updated.
* [X] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST='./huaweicloud/services/acceptance/cph' TESTARGS='-run=TestAccDatasourcePhoneFlavors_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cph -v -run=TestAccDatasourcePhoneFlavors_basic -timeout 360m -parallel 4
=== RUN   TestAccDatasourcePhoneFlavors_basic
=== PAUSE TestAccDatasourcePhoneFlavors_basic
=== CONT  TestAccDatasourcePhoneFlavors_basic
--- PASS: TestAccDatasourcePhoneFlavors_basic (10.73s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cph       10.770s
```

### CODE REVIEW

File: docs/data-sources/cph_phone_flavors.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+# huaweicloud_cph_phone_flavors
+
+Use this data source to get available flavors of CPH phone.
+
+## Example Usage
+
+```
+data "huaweicloud_cph_phone_flavors" "flavor" {
+}
```
Comment (Lance52259) [inline docs/data-sources/cph_phone_flavors.md]: That is too simple

Comment (hmbig2) [reply docs/data-sources/cph_phone_flavors.md]: ..... add `type = 1`

File: huaweicloud/services/cph/data_source_huaweicloud_cph_phone_flavors.go [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+			},
+			"vcpus": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Description: `The vcpus of the CPH phone.`,
+			},
+			"memory": {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Description: `The ram of the CPH phone in MB.`,
```
Comment (ShiChangkuo) [inline huaweicloud/services/cph/data_source_huaweicloud_cph_phone_flavors.go:60]: please check the unit is MB or GB?

**Pull request merged**